docs/shared/ww3.yaml

```
@@ -0,0 +1,10 @@
+ww3:
+  namelist:
+    template_file: /path/to/ww3/ww3_shel.nml.IN
+    template_values:
+      input_nml:
+        input_forcing_winds: "C"
```

```suggestion
    template_values:
      input_forcing_winds: "C"
```
From the point of view of `uwtools`, the input template file is flat text, just containing Jinja2 expressions that need to be rendered. So there's no use for structure under `template_values:` except for a key-value mapping.

Given how the `namelist_file()` task method in the `WaveWatchIII` driver is written, I assume that a config structured like this would fail if given as input in a CLI command like `uw ww3 namelist_file --config-file config.yaml` where this is `config.yaml`. It's probably worth creating a simple `ww3_shel.nml.IN` file and doing that test before merging this.

---

Hmmm - so this is interesting because the entry being edited here in `ww3_shel.nml` is 
```
&INPUT_NML
  INPUT%FORCING%WINDS         = 'H'
```

---

What we'd expect to see in the input template file would be something like
```
&INPUT_NML
  INPUT%FORCING%WINDS = {{ input_forcing_winds }}
```
right?

---

Ha, yes - there are many places in this driver where there are similar topics with different issues, I think this is what got me here